src/features/wallet/SideBarMenu.tsx

```diff
@@ -217,6 +143,79 @@ export function SideBarMenu({
   );
 }
 
+function AccountSummary({ account, address }: { account: AccountInfo; address: Address }) {
```

Comment:
Not much is new here, mostly just some component refactoring while I was implementing the new transfer schema